library-valcon-internship/src/models/Book.ts

```diff
@@ -8,6 +8,6 @@ export default interface Book {
   Quantity: number
   Available: number
   Cover: string
-  PublishDate: string
+  PublishDate: Date
```

Comment:
Optional: If you have time left you can introduce some date-time library to use instead of JS date, like DayJS DateFNS or Luxon.
https://medium.com/@drevets/exploring-unexpected-behavior-with-javascript-date-objects-edaec2b527f9